src/NServiceBus.Core/Persistence/Msmq/SubscriptionStorage/MsmqSubscriptionStorageQueue.cs

```diff
@@ -51,14 +52,15 @@ public void TryReceiveById(string messageId)
         {
             try
             {
-                q.ReceiveById(messageId, MessageQueueTransactionType.None);
+                queue.ReceiveById(messageId, MessageQueueTransactionType.None);
```

Comment:
I think we talked about adding a comment here to make it clear that `None` is not a mistake but intention here?